### PR TITLE
Add GQL status to the `DEPRECATION` category

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-153.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-153.surge.sh/status-codes/5/notifications/all-notifications)

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 1]
```diff
@@ -1275,46 +1658,107 @@ Update the option `indexProvider` with the value `text-2.0`.
...
-====
 
-.Using id() function
-====
+Returned GQLSTATUS code::
+01N01
+
+Returned Status Description::
+warn: feature deprecated with replacement.
+`unsupported.dbms.shutdown` is deprecated.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1724]: I have a question here. We use non-existing procedure in this example, but in the next example we're showing a real use case with `id()` function which has been replaced with `elementid()`.
So my question is: will it be better to change `unsupported.dbms.shutdown` to (for ex.) `cdc.query`, the real deprecated procedure? In this case, we can say that `cdc.query` was deprecated and replaced by `db.cdc.query`.  Will it make the example more visual and helpful?

Comment (Lojjs) [reply modules/ROOT/pages/notifications/all-notifications.adoc:1724]: Yes, absolutely, a real example is better. I think at the time we wrote these docs, we did not have any built-in procedure which was actually deprecated so therefore we made one up. Feel free to update the example

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-153.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-153.surge.sh/status-codes/5/notifications/all-notifications)

**Review by Lojjs — CHANGES_REQUESTED** [review]
Two of them looks wrong to me, otherwise great work!

**Review by Lojjs — APPROVED** [review]

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 2]
```diff
@@ -1114,19 +1372,75 @@ MATCH (a)-[:A|:B|:C]-() RETURN *
...
+----
+MATCH (a)-[:A|:B|:C]-() RETURN *
+----
+
+Returned GQLSTATUS code::
+01N01
+
+Returned Status Description::
+warn: feature deprecated with replacement.
+`:A:|B` is deprecated.
```
[the remark this answers is not in the document]
Comment (renetapopova) [reply modules/ROOT/pages/notifications/all-notifications.adoc:1400]: @Lojjs, this looks like a bug if it is the actual returned message.

**Review by renetapopova — COMMENTED** [review]
I added a question for @Lojjs. The rest of my suggestions I'll push directly into the PR.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-153.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-153.surge.sh/status-codes/5/notifications/all-notifications)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-153.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-153.surge.sh/status-codes/5/notifications/all-notifications)

File: modules/ROOT/pages/notifications/all-notifications.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (Lojjs) [reply modules/ROOT/pages/notifications/all-notifications.adoc]: I checked in the a test in the codebase and I think this is just a mistake in docs. For the query above the statusDescription will be "warn: feature deprecated with replacement. `:A|:B|:C` is deprecated. It is replaced by `:A|B|C`."

Good catch!

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-153.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-153.surge.sh/status-codes/5/notifications/all-notifications)

**Review by Lojjs — APPROVED** [review]

**Pull request merged**